web/src/graph.js

```diff
@@ -129,12 +130,22 @@ function stripUnwantedMilliseconds(data){
   });
 }
 
+function normalizeEndpoint(data){
+  return data.map(d => {
+    if(Object.hasOwn(d, 'endpoint')) {
+      return d;
+    }
+    d.endpoint = d.channel;
+    return d;
```

Comment:
I thought I saw code in the upstream side that already accounted for this? Should be fine if only unnecessary? Why not emit a little breadcrumb (println) in the implicit `else` so that we can look across some runs and if we never see the crumb we can remove this?

Reply:
Oh sorry, this is in the UI code, which could be fiddling with non-updated code. Perfect. My bad. This rules.